internal/driver/fetch_test.go

```diff
@@ -193,11 +193,31 @@ func TestFetch(t *testing.T) {
 		source, execName string
 	}
 
-	for _, tc := range []testcase{
+	ts := []testcase{
 		{path + "go.crc32.cpu", ""},
 		{path + "go.nomappings.crash", "/bin/gotest.exe"},
 		{"http://localhost/profile?file=cppbench.cpu", ""},
-	} {
+	}
+
+	if runtime.GOOS != "windows" {
```

Comment:
Could you add a short comment to explain what case we are testing and why Windows is skipped? Something like "Test that paths with a colon character are recognized as file paths if the file exists, rather than as a URL. We have to skip this test on Windows since the colon char is not allowed in Windows paths." (wrap the comment at 80 columns as necessary)